joint:
  home: [0.733216, -2.303996, 2.222002, -3.058677, -0.733774, 3.14139]
  pre_dispense: [-2.8034, -2.2579, 2.1997, -2.6269, -0.3113, 2.6590]
  pre_sense: [1.9334, -2.2579, 2.1997, -2.6269, -0.3113, 2.6590]

cartesian:
  positions:
    1: [0.31, -0.475, 0.91, 0.7071068, 0.0, 0.0, 0.7071068]
    2: [0.16, -0.475, 0.91, 0.7071068, 0.0, 0.0, 0.7071068]
    3: [0.01, -0.475, 0.91, 0.7071068, 0.0, 0.0, 0.7071068]
    4: [-0.14, -0.475, 0.91, 0.7071068, 0.0, 0.0, 0.7071068]
    5: [-0.29, -0.475, 0.91, 0.7071068, 0.0, 0.0, 0.7071068]

    6: [0.31, -0.475, 0.665, 0.7071068, 0.0, 0.0, 0.7071068]
    7: [0.16, -0.475, 0.665, 0.7071068, 0.0, 0.0, 0.7071068]
    8: [0.01, -0.475, 0.665, 0.7071068, 0.0, 0.0, 0.7071068]
    9: [-0.14, -0.475, 0.665, 0.7071068, 0.0, 0.0, 0.7071068]
    10: [-0.29, -0.475, 0.665, 0.7071068, 0.0, 0.0, 0.7071068]

    11: [0.31, -0.475, 0.42, 0.7071068, 0.0, 0.0, 0.7071068]
    12: [0.16, -0.475, 0.42, 0.7071068, 0.0, 0.0, 0.7071068]
    13: [0.01, -0.475, 0.42, 0.7071068, 0.0, 0.0, 0.7071068]
    14: [-0.14, -0.475, 0.42, 0.7071068, 0.0, 0.0, 0.7071068]
    15: [-0.29, -0.475, 0.42, 0.7071068, 0.0, 0.0, 0.7071068]

  home: [0.01, -0.27, 0.61, 0.707, 0, 0, 0.707]
  # dispense: [-0.4698, 0.0, 0.2950, 0.5, -0.5, -0.5, 0.5]
  # pre_dispense: [-0.2698, 0.0, 0.0, 0.6950, 0.5, -0.5, -0.5, 0.5]
  sense: [0.5244, -0.2376, 0.4620, 0.5050, 0.4831, 0.5050, 0.5064]
  pouring:
    regular:
      corner: [-0.300, -0.030, 0.510, 0.671, -0.613, -0.414, 0.048]
      edge: [-0.385, 0.240, 0.510, 0.910, -0.324, -0.109, 0.235]
    liquid:
      corner: [-0.265, -0.03, 0.460, 0.633, -0.645, -0.421, 0.082]
